tests/functional/tnf/test_tnf_add_capacity.py

```
@@ -0,0 +1,124 @@
+        """
+        Add capacity to TNF cluster by creating new local PVs
+        from available unused disks and expanding the StorageCluster.
+        """
+        logger.test_step("Get current OSD count and deviceset count")
+        existing_osd_count = get_osd_count()
+        existing_deviceset_count = get_deviceset_count()
+        existing_osd_pods = get_osd_pods()
+        existing_osd_pod_names = [pod.name for pod in existing_osd_pods]
+        logger.info(
+            f"Current state: {existing_osd_count} OSDs, "
+            f"deviceset count={existing_deviceset_count}"
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)